db/migrate/20210923070806_add_star_rating_to_space.rb

```diff
@@ -0,0 +1,5 @@
+class AddStarRatingToSpace < ActiveRecord::Migration[6.1]
+  def change
+    add_column :spaces, :star_rating, :integer, null: false, default: 0
```

Comment:
Star rating should be a `:decimal` number, (shown with one decimal) - not an `:integer`, so we can show ratings like `4.2`

With the added null value mentioned earlier, I think it might look something like this 🤷 

```ruby
add_column :spaces, :star_rating, :decimal, scale: 1, null: true, default: null 
```

Reply:
As mentioned earlier it makes more sense to me to use zero instead of null. But changing it to decimal or float is definitely something we can do if you think we need it for aggregate